commands/cli.go

```diff
@@ -404,3 +395,85 @@ func confirmationPrompt(prompt string, maxPrompts int) (bool, error) {
 
 	return false, nil
 }
+
+func awsInspect(c *cli.Context) error {
```

Comment:
Why doesn't this use the `Query` interface?

Reply:
It didn't occur to me to use the Query interface here. My thinking was to first: 
1. Create a command that has the same semantics as the `aws` / nuke command, but that would never be destructive. It made sense to me from a maintainability perspective for the code for `inspect-aws` to closely resemble that for `aws` - since the `aws` and `inspect-aws` commands accept a common subset of flags, and work in similar ways but with different goals 
2. Separately, allow for cloud-nuke to be used as a library for convenient scripting against the resource counts in a given region. In this context, the command line flags are usable but not ideal - I was thinking we wanted an interface closer to Terratest here.

Reply:
I think the CLI and the library should share as much of the code as possible. Otherwise, we are maintaining the same functionality in two places, which can lead to drift.

E.g., what if we expose an additional parameter in the CLI, but not in Query? Users would be used to the CLI, but then get confused when they want to then use it in Query. Or vice versa. It would add friction to duplicate the functionality in both places.

OTOH, if the CLI used the `Query` interface, then exposing new parameters can be implemented in one place, and then exposed as a pass through in the CLI.

Looking at the code, I don't really see anything unique about the CLI interface that prevents usage of the `Query` interface, which was why I posed the question.

Reply:
Gotcha. Thanks for that explanation - makes sense to me. 

I'll update the command to use the same Query interface as the library.

Reply:
Done in c7cfc4f!